samples/unit-tests/indicator-rsi/recalculations/demo.js

```diff
@@ -150,4 +147,81 @@ QUnit.test('Test RSI calculations on data updates.', function (assert) {
         ],
         'Correct values after point.remove()'
     );
+    // Enhancement #14930, RSI Works with line series type
+    chart.addSeries({
+        data: [
+            443389,
+            440902,
+            441497,
+            436124,
+            443278,
+            448264,
+            450955,
+            454245,
+            458433,
+            460826,
+            458931,
+            460328,
+            456140,
+            462820,
+            462820,
+            460028,
+            460328,
+            464116,
+            462222,
+            456439,
+            462122,
+            462521,
+            457137,
+            464515,
+            457835,
+            453548,
+            440288,
+            441783,
+            442181,
+            445672,
+            434205,
+            426628,
+            431314
+        ],
+        id: 'line-series',
+        yAxis: 0
+    });
```

Comment:
You could set the `redraw` flag here to `false` to improve the performance (the chart will be redrawn within the next `chart.addSeries()` method call.

Reply:
I switched from creating new series to updating the previous series.